library/pom.xml

```diff
@@ -372,9 +375,37 @@
 							</execution>
 						</executions>
 					</plugin>
+
 				</plugins>
 			</build>
 		</profile>
+		<profile>
+			<!-- These plugins can't be part of the standard build -->
+			<id>ckeditor-extras</id>
+			<activation>
+				<activeByDefault>false</activeByDefault>
+				<property>
+					<name>ckeditor-extras</name>
+				</property>
+			</activation>
+			<properties>
+				<ckeditor-extra-plugins>,a11ychecker</ckeditor-extra-plugins>
```

Comment:
I think this property should still be changed to {ckeditor-a11y-extra-plugins} ?